src/pages/AppDetail.svelte

```diff
@@ -73,8 +74,15 @@
   >
     <Tab key="metrics">Metrics</Tab>
     <Tab key="pings">Pings</Tab>
+    {#if app.labels && app.labels.length}
+      <Tab key="labels">Labels</Tab>
```

Comment:
Personally I find "Features", instead of "Labels", more idiomatic. I think calling these "labels" is too general and it could be confusing at first to understand what the labels mean, as we also use labels for the glean libraries, item status, etc. The convention for these labels in the fenix repo uses `Features:` as a prefix, so maybe we should call these labels "Features" too?

Reply:
Yeah, I think it's not going to be the same for every application. But maybe we can make the user-visible name we show to people configure. "Features" definitely makes the most sense for Fenix. Will investigate.

Reply:
Ok, I'm just going to call them "Features" for this iteration, since this only applies to Fenix.

Reply:
👍